evalml/tests/automl_tests/test_dask_engine.py

```diff
@@ -230,6 +243,39 @@ def test_cancel_job(self):
         pipeline_future.cancel()
         assert pipeline_future.is_cancelled
 
+    def test_dask_sends_woodwork_schema(self):
+        X, y = self.X_y_binary
+        engine = DaskEngine(client=self.client)
+
+        X.ww.init(logical_types={0: "Categorical"}, semantic_tags={0: ['my cool feature']})
+        y = ww.init_series(y)
+
+        new_config = AutoMLConfig(ensembling_indices=automl_data.ensembling_indices,
+                                  data_splitter=automl_data.data_splitter,
+                                  problem_type=automl_data.problem_type,
+                                  objective=automl_data.objective,
+                                  additional_objectives=automl_data.additional_objectives,
+                                  optimize_thresholds=automl_data.optimize_thresholds,
+                                  error_callback=automl_data.error_callback,
+                                  random_seed=automl_data.random_seed,
+                                  X_schema=X.ww.schema,
+                                  y_schema=y.ww.schema)
+
+        # TestSchemaCheckPipeline will verify that the schema is preserved by the time we call
+        # pipeline.fit and pipeline.score
+        pipeline = TestSchemaCheckPipeline(component_graph=["One Hot Encoder", "Logistic Regression Classifier"],
+                                           parameters={"Logistic Regression Classifier": {"n_jobs": 1}},
+                                           X_schema_to_check=X.ww.schema, y_schema_to_check=y.ww.schema)
```

Comment:
Great call adding this test, and I like your solution of sticking the assertions in the pipeline methods and adding a comment explaining that here.